utils/userData.ts

```diff
@@ -49,3 +48,47 @@ export async function UpdateUser(user: {
     throw error;
   }
 }
+
+export async function InsertDeposit(deposit: {
```

Comment:
i think would be more uniforme if `insertDeposit`? we already have `GetUser` in there, tho-- which i think is different from `getUser`. this is bad, but I don't care that much.

Reply:
getUser is an internal function, doesn't work as is, it is called by GetUser. 
Considering that the callable function is GetUser, I went with InsertDeposit.
This is also the naming convention used in backend languages (such as Go). camel case for internal, pascal for external